cassandra_nodetool/MANIFEST.in

```diff
@@ -0,0 +1,6 @@
+include README.md
```

Comment:
You don't have to touch this newly created file. https://github.com/DataDog/integrations-core/pull/1727/files